### DISCUSSION TITLE
Any update for segment NK1?

### DISCUSSION THREAD

**Discussion by QuiVN** [senaranya/HL7#84] [General]
I don't see the PHP file for the NK1 segment, will there be future updates?

**Comment by senaranya** [comment]
I can add it sometime over the next few days. If you need it immediately feel free to create one in a Pull Request.

**Comment by QuiVN** [comment]
Many thanks for the answer. Please let me know when there is a new update. Thanks you.

**Comment by senaranya** [comment]
Added segment NK1 and created a new release 2.1.7

**Reply by QuiVN** [reply]
Thanks a lot for your quick update. Wishing you a new day full of energy and happiness. From Vietnam <3 !!!

**Reply by senaranya** [reply]
No problem at all. Wish you a good day too :)
